packages/page-staking/src/index.tsx

```diff
@@ -125,10 +125,10 @@ function StakingApp ({ basePath, className = '' }: Props): React.ReactElement<Pr
       />
       <Switch>
         <Route path={`${basePath}/payout`}>
-          <Payouts
+          {areAccountsLoaded && !isLoadingStashes && <Payouts
```

Comment:
I actually just did something similar here - https://github.com/polkadot-js/apps/pull/5674/files#diff-f92441f5f0aafb2ba6f441b16b386be924463f6c3fd4d373a41ee5da1e1f4e2dR84

(basically in that case the route is not available since we don't have any stashes)

The right way is indeed, like above to hide the tab, not to hide the content since then it appears unresponsive for however long it takes.

Reply:
sure, definitely better/proper, thx